srcs/app/server/WebServer.cpp

```diff
@@ -21,8 +25,6 @@ void	WebServer::Run() {
 				fdSets.getWriteSet(), NULL, NULL);
 		if (ready_sockets < 0) {
 			throw std::runtime_error(std::strerror(errno));
-		} else if (ready_sockets == 0) {
```

Comment:
Why was it a bad idea to throw an error when select returned zero as well?

Reply:
It was not necessarily a bad idea, but with a `NULL` timeout, `select()` will never return 0.

From the manual of `select()`:

>   If timeout is specified as NULL, select() blocks
indefinitely waiting for a file descriptor to become
ready.